packages/peregrine/lib/talons/Header/storeSwitcher.gql.js

```diff
@@ -11,11 +11,20 @@ export const GET_STORE_CONFIG_DATA = gql`
     }
 `;
 
-export const GET_URL_RESOLVER_DATA = gql`
-    query getUrlResolverData($url: String!) {
-        urlResolver(url: $url) {
-            id
+export const GET_ROUTE_DATA = gql`
+    query getRouteData($url: String!) {
+        route(url: $url) {
             type
+            ... on CmsPage {
+                identifier
```

Comment:
Might want to pick up the changes coming from here
https://github.com/magento/pwa-studio/pull/3580/files